test/parallel/test-assert.js

```diff
@@ -750,7 +750,8 @@ common.expectsError(
   assert.equal(assert.deepEqual, assert.deepStrictEqual);
   assert.equal(assert.notEqual, assert.notStrictEqual);
   assert.equal(assert.notDeepEqual, assert.notDeepStrictEqual);
-  assert.equal(Object.keys(assert).length, Object.keys(a).length);
+  // strict doesn't expose promises
```

Comment:
Why not?

Reply:
In this PR, you have currently (all synchronous):
- `require('assert').equal`
- `require('assert').strict.equal`
- `require('assert').promises.equal`
- `require('assert').promises.strict.equal`

I wasn't sure it worth providing `require('assert').strict.promises`. Do you think it worth it?

Reply:
Copy that